### PR TITLE
Fix/navigation menu

### PR DESCRIPTION
**Pull request by chovy**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -69,4 +42,13 @@
   </mat-card>
   <router-outlet></router-outlet>
   <app-messages></app-messages>
+  <footer>
+    <a href="{{explorerHost()}}" class="sugar">
+      <span>Blocks</span>
```
Comment (treeder) [inline src/app/app.component.html]: Can you call this "Explorer" or "Block Explorer"

**Comment by chovy** [comment]
screenshot 2018-06-20 15 37 18

**Comment by chovy** [comment]
fixed spacing issue.

screenshot 2018-06-21 00 12 16

**Comment by jmank88** [comment]
What about some `margin-top` on the footer links? Maybe as much as is between each of those 'cards' or whatever?
Maybe the links would look nice centered as well?

**Comment by chovy** [comment]
screenshot 2018-06-21 09 45 10

**Comment by treeder** [comment]
Much better.

**Pull request merged**